### PR TITLE
refactor: restructure with lockup dir

### PR DESCRIPTION
**Pull request by andreivladbrg**
Closes https://github.com/sablier-labs/v2-deployments/issues/10, #11, #13 

> [!IMPORTANT]  
> The merge of this PR should be synchronized with renaming this repo to `deployments` (removing "V2") and updating the documentation in places such as [this](https://github.com/sablier-labs/docs/blob/5c96e045093887643d24eb837b4e8385abf8d177/docs/contracts/v2/02-deployments.md?plain=1#L50-L53)

### CODE REVIEW

**Review by smol-ninja — APPROVED** [review]
Renaming this repo to `deployment` can be synchronised with renaming `v2-core` to `lockup`. So, feel free to merge this and open another issue about renaming repo.

**Comment by andreivladbrg** [comment]
> So, feel free to merge this and open another issue about renaming repo.

we need to also update the hyperlinks in docs

**Comment by smol-ninja** [comment]
Right. So should we mark it as draft then?

**Comment by andreivladbrg** [comment]
> Right. So should we mark it as draft then to prevent accidental merging of this PR

we could, but some clarifications, doesn't "draft" mean that the work for the individual PR is not finished? 
the case here is that for deployments PR is finished, but we need to updated other repos

**Comment by smol-ninja** [comment]
As per https://github.blog/news-insights/product-news/introducing-draft-pull-requests/, Draft PR can be used to send various signals one of which is to block merging the pull request.

File: flow/v1.0.0/README.md
Comment (smol-ninja) [inline flow/v1.0.0/README.md:36]: We should add similar table for ZKsync as well.

**Review by smol-ninja — CHANGES_REQUESTED** [review]
Here is another review after merging https://github.com/sablier-labs/v2-deployments/pull/14.

1. ZKsync broadcasts and ZKsync artifacts are missing. ZKsync artifacts should be different because of hardhat requirement (similar to lockup directory).
2. We should update the [Scripts section](https://github.com/sablier-labs/v2-deployments/tree/refactor/restructure-repo?tab=readme-ov-file#scripts) in the README file as the bash scripts are no longer used to make deployments.
3. Link to Flow npm package was missing from README which I have added in https://github.com/sablier-labs/v2-deployments/pull/12/commits/61e0b2f084871047f0b4d56bbff86e1e301a155f.

**Comment by andreivladbrg** [comment]
> ZKsync broadcasts 

unfortunetely i didn't have them this time, i don't know why

> ZKsync artifacts are missing

will add them

**Review by smol-ninja — APPROVED** [review]
LGTM.

**Comment by smol-ninja** [comment]
Should we merge it @andreivladbrg given we are going live in a few hours? We will then have to update the links in docs as well.

**Pull request merged**
